snacks/tests.py

```diff
@@ -1,3 +1,66 @@
+from django.contrib.auth import get_user_model
 from django.test import TestCase
+from django.urls import reverse
+
+from .models import Snack
+
 
 # Create your tests here.
+
+
+class SnackTests(TestCase):
+    def setUp(self):
+        self.user = get_user_model().objects.create_user(
+            username="tester", email="tester@email.com", password="pass"
+        )
+
+        self.snack = Snack.objects.create(
+            name="pickle",
+            reviewer=self.user,
```

Comment:
The idea of creating a test snack is correct. There's a bug here that's tripping up the unit tests: All three keys (title, description, and purchaser) must match the values in models.py for a test snack to be successfully added.